style.css

```diff
@@ -412,12 +412,111 @@ header {
 }
 
 .skills {
-  /* margin-left: 145px;
-  margin-right: 115px; */
   padding-right: 40px;
   margin-top: 12px;
 }
 
 .frameworks {
   padding-left: 12px;
 }
+
+.project1:hover,
+.project2:hover {
+  background-color: #daa884;
+  cursor: pointer;
+}
+
+#resume:hover {
+  background-color: #daa884;
+  cursor: pointer;
+}
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+}
+
+.contact h1 {
+  font-family: 'Inter', sans-serif;
+  font-weight: bolder;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 120px 24px 105px;
+  color: #172b4d;
+  text-align: center;
+  min-height: 216px;
+  justify-content: center;
+}
+
+.form-1 {
+  display: flex;
+  flex-direction: column;
+  margin-left: 1.5rem;
+  margin-right: 1.5rem;
+}
+
+#name,
+#email,
+#textarea-1 {
+  padding: 12px;
+  border: none;
+  margin-bottom: 14px;
+  margin-top: 39px;
+}
+
+#textarea-1 {
+  background-color: #fbf8f7;
+  color: black;
+}
+
+input::placeholder {
+  color: #979493;
+}
+
+#textarea-1::placeholder {
+  color: black;
+  font-weight: 400;
+}
+
+#get-in-touch {
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  background-color: #ff6b00;
+  color: white;
+  margin: 36px 134px 134px;
+  padding: 12px;
+  border: none;
+}
+
+#get-in-touch:hover {
+  background-color: #daa884;
+  cursor: pointer;
+}
```

Comment:
Nice job implementing the hover effect :+1: However, It should match the one presented in the template. You can introduce the following declarations in order to get the desired effect: 

```
  background-color: #FFF5E1;
  color:#ff6b00 ;
  border: 1px solid #ff6b00;
```

You should also implement the effect for the disabled and pressed states for the buttons. 

![Screenshot from 2023-01-25 21-09-42](https://user-images.githubusercontent.com/93520178/214678615-30291c2e-2a91-4339-b8d5-ba2644224fd2.png)